src/ngraph/op/constant.hpp

```diff
@@ -442,6 +442,8 @@ namespace ngraph
         class NGRAPH_API ScalarConstantLikeBase : public Constant
         {
         public:
+            static constexpr NodeTypeInfo type_info{"ScalarConstantLikeBase", 0};
```

Comment:
I think there was at one time a reason for `ScalarConstantLikeBase` independent of `ScalarConstantLike` but I don't see one now. Can we just merge them into `ScalarConstantLike` without breaking transformers?